tests/parser/parser.cpp

```diff
@@ -653,25 +653,32 @@ TEST(Parser, encapsulationWithNoComponentAttribute)
 
 TEST(Parser, encapsulationWithNoComponentRef)
 {
+    const std::vector<std::string> expectedErrors = {
+        "Encapsulation in model 'model_name' has an invalid child element 'component_free'.",
+    };
+
```

Comment:
This ought to be replaced with:

```
    const std::vector<std::string> expectedErrors = {
        "Encapsulation in model 'model_name' has an invalid child element 'component_free'."};
```

which is what we do elsewhere in the code.

Reply:
We have a mix of this stuff.  I prefer the what is there that is why I have done it this way and clang-tidy is happy with this.

Reply:
I don't like trailing commas, but I am not happy either with the way it gets formatted by ClangFormat (when there is no trailing comma). So, I am happy with either in the end. Just want consistency. :)

Reply:
I'm getting used to them, as I prefer them over the alternative.

Reply:
Fine by me. Will just need to be consistent (added a bullet point to that respect to issue #402).